packages/openapi-fetch/test/types.test.ts

```diff
@@ -280,4 +286,179 @@ describe("types", () => {
       assertType<Response>({ error: "default application/json" });
     });
   });
+
+  describe("OpenApiStatusToHttpStatus", () => {
```

Comment:
It is super nice to see unit tests for this helper!

I was wondering: IIUC the second argument (`AllStatuses`) comes directly from the response object keys. In this case, it could contain wildcards (e.g. `"2XX"`). Would it make sense to add tests for these as well? (I have tried pointing out places below where I think this might affect the tests).

@drwpow self note to maintainers: we should probably set up the infrastructure for `openapi-typescript-helpers` to have tests :)